packages/node-integration-tests/suites/express/sentry-trace/baggage-user-id/test.ts

```diff
@@ -3,7 +3,8 @@ import * as path from 'path';
 import { getAPIResponse, runServer } from '../../../../utils/index';
 import { TestAPIResponse } from '../server';
 
-test('Includes user_id in baggage if sendDefaultPii is set to true', async () => {
+// Skipping this test because right now we're not including user_id at all
+test.skip('Includes user_id in baggage if sendDefaultPii is set to true', async () => {
```

Comment:
Since, as discussed in the meeting earlier, we're not going to be using `sendDefaultPII` to control this (now or ever), we should pull it from any tests it's in. 

We can leave this test, maybe, since _something_ will eventually control that, but if so I'd probably do something like
```suggestion
// TODO: Skipping this test because right now we're rethinking the mechanism for including such data
test.skip('Includes user_id in baggage if <optionTBA> is set to true', async () => {
```

and add a similar TODO to the companion file which sets up the test.

Reply:
Done [c8aac6e](https://github.com/getsentry/sentry-javascript/pull/5363/commits/c8aac6e8475946c1870666a15d51489addbf5639)